src/main/resources/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobProperty/config.jelly

```diff
@@ -31,9 +31,15 @@ THE SOFTWARE.
     <j:invokeStatic var="triggers" className="hudson.triggers.Trigger" method="for_">
         <j:arg value="${it}" type="hudson.model.Item" />
     </j:invokeStatic>
-    <f:descriptorList title="${%Build Triggers}"
+    <f:descriptorList field="triggers"
+                      title="${%Build Triggers}"
                       descriptors="${triggers}"
                       instances="${it.triggers}">
         <d:invokeBody />
     </f:descriptorList>
+    <f:block>
+        <div class="section-header">
+            ${%Other Properties}
```

Comment:
Hmm, clumsy, inconsistent with freestyle projects, and fragile (you have no way of knowing for sure where this property will appear). I wonder if we could somehow make this property appear last in the list instead, using `ordinal`. Also a hack but at least would look better.

Alternately we could have a proper section for **Properties** (?) and then have **Build Triggers** be an indented block rather than a section…though this would again be inconsistent with traditional project types.